### PR TITLE
Add turkish locales

### PR DESCRIPTION
**Pull request by mikropsoft**

### CODE REVIEW

File: app/src/main/res/values-tr/strings.xml [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+    <string name="show_variables_sum">Tüm çekirdek parametrelerini bir listede görüntüleme ve düzenleme</string>
+    <string name="sysctl_operations">Sysctl işlemleri</string>
+    <string name="read_from_file">Dosyadan değerleri oku</string>
+    <string name="read_from_file_sum">Seçilen bir .conf veya .json dosyasından parametre yüklemek için bu seçeneği kullanın</string>
+    <string name="show_favorites">Favorileri göster</string>
+    <string name="show_favorites_sum">Bu listeden en sevdiğiniz çekirdek parametrelerine erişin</string>
+    <string name="exit">Çıkış</string>
+    <string name="kernel_params">Çekirdek parametreleri</string>
+    <string name="browse_variables">Değişkenlere göz at</string>
+    <string name="browse_variables_sum">Belirli bir çekirdek parametresi için /proc/sys'ye göz atın</string>
```
Comment (Lennoard) [inline app/src/main/res/values-tr/strings.xml:20]: This string resource contains an unescaped apostrophe (`'`)

Same for lines [92 and 93](https://github.com/Lennoard/SysctlGUI/pull/62/files#diff-11548b2158b255e354ac7ab1cdc4dc4bf1d0cb372bbf1425508d86d35b69ed50R92-R93)

Suggestion: fix apostrophe escaping by using a `\` character before each.

File: app/src/main/res/values-tr/params_info.xml [hunk 2]
```diff
@@ -0,0 +1,55 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <!-- VM -->
+    <string name="admin_reserve_kbytes">Cap_sys_admin özelliğine sahip kullanıcılar için ayrılması gereken sistemdeki boş bellek miktarı</string>
+    <string name="block_dump">Sıfır olmayan bir değere ayarlandığında blok G/Ç hata ayıklamasını etkinleştirir</string>
+    <string name="compact_memory">Yalnızca CONFIG_COMPACTION ayarlandığında kullanılabilir. Dosyaya 1 yazıldığında, tüm bölgeler, mümkün olduğunda bitişik bloklarda boş bellek mevcut olacak şekilde sıkıştırılır.</string>
+    <string name="compact_unevictable_allowed">Yalnızca CONFIG_COMPACTION ayarlandığında kullanılabilir. 1 olarak ayarlandığında, sıkıştırmanın sayfaların sıkıştırılması için kaçınılmaz lru'yu (kilitli sayfalar) incelemesine izin verilir. Sıkıştırmanın önlenemeyen sayfaları hareket ettirmesini önlemek için 0 olarak ayarlayın. Varsayılan değer 1'dir.</string>
```
Comment (Lennoard) [inline app/src/main/res/values-tr/params_info.xml:7]: This string resource contains an unescaped apostrophe (`'`).
Same for lines 10, 13, 15, 16, 20, 21, 22, 27, 29, 30 and 32.

Suggestion: fix apostrophe escaping by using a `\` character before each.

**Comment by Lennoard** [comment]
@mikropsoft thank you very much for your contribution. Unfortunately, some strange thing is happening that prevents the building the app with your additions. It is most likely related to illegal unicode characters the compiler can't allow.

- `res/values-tr/strings.xml` line 23
- `res/values-tr/params_info.xml` line 30

Suggestion: simplify characters in these lines.

**Review by Lennoard — CHANGES_REQUESTED** [review]

**Review by Lennoard — APPROVED** [review]
Looking good!

**Comment by mikropsoft** [comment]
> Looking good!

Good luck 💯

**Pull request merged**